test/models/vrp_test.rb

```diff
@@ -304,5 +304,22 @@ def test_no_lapse_in_relation
       Models::Vrp.filter(vrp)
       refute_empty vrp[:relations] # do not reject even if no lapse, lapse is not mandatory
     end
+
+    def test_original_skills_and_skills_are_equal_after_create
+      vrp = VRP.basic
+      vrp[:vehicles].first[:skills] = [['skill_to_output']]
+      vrp[:services].first[:skills] = ['skill_to_output']
+
+      created_vrp = Models::Vrp.create(vrp)
+      assert_equal 1, created_vrp.services.first.skills.size
+      assert_equal created_vrp.services.first.original_skills.size, created_vrp.services.first.skills.size
+    end
+
+    def test_geometry_provided
+      vrp = VRP.toy
+      vrp[:configuration][:restitution] = { geometry: %i[partitions points polylines], geometry_polyline: true }
+      Models::Vrp.ensure_retrocompatibility(vrp)
+      assert_equal %i[polylines], vrp[:configuration][:restitution][:geometry]
```

Comment:
This should raise. Geometry is a new field, if someone is starting to use the new field, they can stop using the old field. 

If only the old field `geometry_polyline: true` is sent, it should continue to work as expected (as if they sent `geometry: %i[polylines]`) but if they sent the two, we should raise like the `solver_parameter`. Otherwise, we are erasing the info sent by the user which might be a mistake so we should fail "loudly"

Reply:
Geometry is not a new field, I changed behaviour because previously if _geometry_ was equivalent to _geometry_polylines_. It is therefore useless to keep both.

Users used to send { geometry: true, geometry_polylines: true } to get what we now call { geometry: [polylines] }

Reply:
You are right. I started like this and then changed it because it was dirty but you are right, we need to know if user is mixing old and new parameters.

This implies : that _geometry_ can be either an array of symbol, or a boolean.
If it is false we just return _geometry_ = [], otherwise we need to keep _true_ value until we get to vrp.create and detect if we used _geometry_polylines_ or not. 

Do you see a better way of doing this ?

Reply:
No, not really. It'll be again `ensure_retrocompatibility`